### PR TITLE
Updated PyPortal Google Calendar example to make better use of PyPort…

### PR DESCRIPTION
**Pull request by makermelissa**
…al library

This is a followup to #2108 and fixes the PyPortal example.

I also increased the size of the QR code, as it was a bit on the small side and my phone wouldn't scan it.

### CODE REVIEW

**Review by TheKitty — APPROVED** [review]

**Comment by brentru** [comment]
@TheKitty  I was tagged for a review and didn't get a chance to check/test the updates melissa did to my original code before it was merged, pls. don't merge if I'm tagged for review&testing.

**Review by brentru — COMMENTED** [review]
LGTM, guide updated

**Pull request merged**
